Homework_4.py

```diff
@@ -0,0 +1,15 @@
+#1. Створити програму, яка буде очікувати введення тексту від користувача і повідомляти —
+#чи є введений текст “числом” чи “словом”.
+#2. Якщо введений текст “число”, програма має також вказати чи є воно парним чи непарним.
+#3. Якщо це “слово”, програма має вказати його довжину.
+
+def type_checker():
```

Comment:
Remove this code from the current PR and create a separate one. Please, take to account some comments below

Reply:
What is the point to create another PR? Do I need to create another branch for it?

Reply:
No, you should make changes here and add as another commit

Reply:
You should create separate branch for every homework, and, as a result, there will be separates Pull Requests